lib/index.js

```diff
@@ -44,7 +44,8 @@ class ElectronDownloader {
   }
 
   get cache () {
-    if (this.opts.cache) return this.opts.cache
+    let cacheLocation = this.opts.cache || process.env.ELECTRON_CACHE
```

Comment:
Hate being that guy 😆 But this should really be `const` 👍

Reply:
@MarshallOfSound Fixed. ESLint rule added to prevent such errors in the future.